Assets/Samples/5 - Initial Prompt/InitialPromptDemo.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+using UnityEngine.UI;
+// ReSharper disable ArrangeObjectCreationWhenTypeEvident - for Unity 2019/2020 support:
+
+namespace Whisper.Samples
+{
+    public class InitialPromptDemo : MonoBehaviour
```

Comment:
Could you move this in first demo? Right now there are already a lot of scenes and it's hard to test and maintain them.